src/components/pages/order/navBar/RightSide.js

```diff
@@ -0,0 +1,51 @@
+import { useState } from "react";
+import styled from "styled-components";
+import { theme } from "../../../../theme";
+import { useParams } from "react-router-dom";
+import UserProfile from "./UserProfile";
+import ToggleButton from "../../../reusable/ToggleButton";
+import { toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+import AdminModeToast from "./AdminModeToast";
+
+export default function RightSide() {
+  const { username } = useParams();

```

Comment:
Ta props `username` que tu fais passer du composant NavbarRightSide à UserProfile
Tu peux directement la consommer là où tu en à besoin en l'occurrence dans ton composant `<UserProfile />` à partir du hook useParams

Reply:
D'accord, super, j'ai cru que ça s'appliquait qu'à la page à laquelle correspond le path sur lequel on arrive quand on se log in ! Fixed

Reply:
Oui, tu peux le faire sans problème ;)